#
#
# patch "libpurple/protocols/oscar/oscar.h"
#  from [8b594b0e56d54fb434bdd19226001450421a89eb]
#    to [6846afcaad3e902e2931f3ed6eea7f8693f4a69a]
#
============================================================
--- pidgin-2.4.3.orig/libpurple/protocols/oscar/oscar.h	8b594b0e56d54fb434bdd19226001450421a89eb
+++ pidgin-2.4.3/libpurple/protocols/oscar/oscar.h	6846afcaad3e902e2931f3ed6eea7f8693f4a69a
@@ -286,6 +286,15 @@ struct _ClientInfo
 	"us", "en", \
 }
 
+#define CLIENTINFO_ICQBASIC_14_34_3096 { \
+	"ICQBasic", \
+	0x010a, \
+	0x0014, 0x0034, \
+	0x0000, 0x0c18, \
+	0x0000043d, \
+	"us", "en", \
+}
+
 #define CLIENTINFO_NETSCAPE_7_0_1 { \
 	"Netscape 2000 an approved user of AOL Instant Messenger (SM)", \
 	0x1d0d, \
@@ -312,14 +321,14 @@ struct _ClientInfo
 #define CLIENTINFO_PURPLE_ICQ { \
 	"Purple/" VERSION, \
 	0x010a, \
-	0x0006, 0x0000, \
-	0x0000, 0x17ab, \
-	0x00007535, \
+	0x0014, 0x0034, \
+	0x0000, 0x0c18, \
+	0x0000043d, \
 	"us", "en", \
 }
 
 #define CLIENTINFO_AIM_KNOWNGOOD CLIENTINFO_AIM_5_1_3036
-#define CLIENTINFO_ICQ_KNOWNGOOD CLIENTINFO_ICQBASIC_14_34_3000
+#define CLIENTINFO_ICQ_KNOWNGOOD CLIENTINFO_ICQBASIC_14_34_3096
 
 typedef enum
 {
